docpages/example_programs/misc/using-emoji.md

```diff
@@ -0,0 +1,9 @@
+\page using-emoji Using Emoji
+
+Need your bot to use an emoji? Then the `dpp::emoji` class is just for you. It can store either a custom or a unicode emoji's name and ID. To use one in a message or when reacting to one, you use its mention. In select menus, it's the emoji object itself. Here's how to use them:
```

Comment:
Tbh, the more I think of `Then the dpp::emoji class is just for you`, the more I think it doesn't work. This comes across that `dpp::emoji` should be used in all instances.

This text should be reworded to state both `dpp::emoji` and `dpp::unicode_emoji` and what the difference is.

The `In select menus, it's the emoji object itself.` part I'd put in a note, people may not read this text fully so a separate note is handy for this.

Reply:
Like I said, this part needs fixing